Source/Eto/Forms/Window.cs

```diff
@@ -118,6 +118,30 @@ protected virtual void OnLocationChanged(EventArgs e)
 		}
 
 		/// <summary>
+		/// Identifier for handlers when attaching the <see cref="OwnerChanged"/> event.
+		/// </summary>
+		EventHandler<EventArgs> OwnerChangedEvent;
+
+		/// <summary>
+		/// Occurs when the <see cref="Owner"/> is changed.
+		/// </summary>
+		public event EventHandler<EventArgs> OwnerChanged
+		{
+			add { OwnerChangedEvent += value; }
+			remove { OwnerChangedEvent -= value; }
+		}
```

Comment:
This should be changed to use the Properties.AddEvent() and Properties.RemoveEvent() so as to remove memory overhead when not handled